### DISCUSSION TITLE
How to un pause account?

### DISCUSSION THREAD

**Discussion by tonydavidx** [vercel/community#4932] [Account Issue]
### Question

My vercel has been paused for exceeding free usage limits. I have deleted the project that caused high usage of free limits.  My account is still paused, I get this message in my dashboard 

_"Your Personal Account has been paused for exceeding free usage limits. Create a Pro Team to continue deploying."_

 what should I do to un pause my account?

**Comment by amyegan** [comment]
Hey Antony. You can find more info about paused accounts at https://vercel.com/guides/why-is-my-account-deployment-blocked

Please check your email for a message with details about the pause and next steps.
